tools/topology/CMakeLists.txt

```diff
@@ -157,6 +157,11 @@ foreach(tplg ${TPLGS})
 
 	add_custom_command(
 		OUTPUT ${output}.tplg
+# - In alsa-utils commit v1.2.2~15-gcbabe7a3f0cc, alsatplg (accidentally?)
+#   dropped the verbosity level; -v became a boolean.
```

Comment:
"accident" at https://github.com/alsa-project/alsa-utils/commit/cbabe7a3f0cc84ecd3#r38034335

Reply:
I'm not sure if I understand. What do you mean by 'bacame a boolean', what other arg do I have to pass instead of '-v', something like '-v true'?

Reply:
I'm not 100% sure what happens when you misconfigure `getopt` but in my testing I observed that `-v whatever` is verbose, including `-v 0`